### DISCUSSION TITLE
Please tell me your favorite athlete?　あなたの好きなアスリートを教えてください．

### DISCUSSION THREAD

**Discussion by nagata-atsuki** [NAIST-SE/AttractingContributors#160] [General]
Any sports is fine
example
2. Rui Hachimura
3. Michael Jeffrey Jordan
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by MatsumuraTaiki** [comment]
Added, please help me too
https://github.com/MatsumuraTaiki/AttractingContributors-Actors/edit/main/README.md

**Comment by ElefunAi** [comment]
I have added Athlete list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Comment by veekay7** [comment]
I have added to the athelete list.
Please help. Cheers.
https://github.com/VantelinLuca/AttractingContributors-WishCountry
